llrt_core/src/module_loader/resolver.rs

```diff
@@ -548,7 +553,7 @@ fn load_package_self(ctx: &Ctx<'_>, x: &str, dir: &str, is_esm: bool) -> Result<
     //    "." + X.slice("name".length), `package.json` "exports", ["node", "require"])
     //    <a href="esm.md#resolver-algorithm-specification">defined in the ESM resolver</a>.
     // 6. RESOLVE_ESM_MATCH(MATCH)
-    if let Ok(path) = package_exports_resolve(&package_json, &name, is_esm) {
+    if let Ok((path, _)) = package_exports_resolve(&package_json, &name, is_esm) {
```

Comment:
We don't yet know if we need the same implementation here. Even if we did, the path is still relative, so adding a prefix at this point would be a mistake.